src/coreclr/nativeaot/Bootstrap/main.cpp

```diff
@@ -106,9 +106,10 @@ void* PalGetModuleHandleFromPointer(void* pointer);
 
 #if defined(HOST_X86) && defined(HOST_WINDOWS)
 #define STRINGIFY(s) #s
+#define LINKER_COMMENT(comment) _Pragma(STRINGIFY(comment))
```

Comment:
Nit: unnecessary diff. Please run runtime-nativeaot-outerloop if you want to keep it, we don't have x86 coverage by default in CI.